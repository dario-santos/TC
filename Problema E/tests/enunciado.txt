11
S -> A
S -> B D e
A -> A e
A -> e
B -> d
B -> C C
C -> e C
C -> e
C -> _
D -> a D
D -> a
