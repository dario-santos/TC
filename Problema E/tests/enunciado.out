NULL(S) = False
NULL(A) = False
NULL(B) = True
NULL(C) = True
NULL(D) = False
FIRST(S) = a d e
FIRST(A) = e
FIRST(B) = d e
FIRST(C) = e
FIRST(D) = a
FOLLOW(S) = #
FOLLOW(A) = # e
FOLLOW(B) = a
FOLLOW(C) = a e
FOLLOW(D) = e
